lib/dao.js

```diff
@@ -1242,22 +1243,22 @@ DataAccessObject._normalize = function(filter) {
   }
   var err = null;
   if ((typeof filter !== 'object') || Array.isArray(filter)) {
-    err = new Error(util.format('The query filter %j is not an object', filter));
+    err = new Error(g.f(util.format('The query filter %j is not an {{object}}', filter)));
     err.statusCode = 400;
     throw err;
   }
   if (filter.limit || filter.skip || filter.offset) {
     var limit = Number(filter.limit || 100);
     var offset = Number(filter.skip || filter.offset || 0);
     if (isNaN(limit) || limit <= 0 || Math.ceil(limit) !== limit) {
-      err = new Error(util.format('The limit parameter %j is not valid',
-        filter.limit));
+      err = new Error(g.f(util.format('The {{limit}} parameter %j is not valid',
+        filter.limit)));
       err.statusCode = 400;
       throw err;
     }
     if (isNaN(offset) || offset < 0 || Math.ceil(offset) !== offset) {
-      err = new Error(util.format('The offset/skip parameter %j is not valid',
-          filter.skip || filter.offset));
+      err = new Error(g.f(util.format('The {{offset/skip}} parameter %j is not valid',
```

Comment:
This string won't be extracted, thus translated.
solution is simply: `util.format` ---> `g.f`